src/emmetHelper.ts

```diff
@@ -169,8 +169,8 @@ function getCurrentWord(document: TextDocument, position: Position): string {
 	}
 }
 
-function removeTabStops(expandedWord: string): string {
-	return expandedWord.replace(/\$\{\d+\}/g, '').replace(/\$\{\d+:([^\}]+)\}/g, '$1');
+function makeCursorsGorgeous(expandedWord: string): string {
+	return expandedWord.replace(/\$\{\d+\}/g, '|').replace(/\$\{\d+:([^\}]+)\}/g, '_$1_') + '|';
```

Comment:
The last cursor should be skipped.
It is not a tab stop once the snippet gets expanded in VS Code.

We should retain the previous `removeTabStops` method and use it for css/scss when setting fitlerText, sortText and label. See Thanks for all the love for this feature! Credits go to @alexandrevez who came up with the idea and PR. (I finally found him on twitter!)